doc/archiving.md

```diff
@@ -0,0 +1,42 @@
+# How to archive a Smart Answer
+
+To prevent a Smart Answer being [registered with Panopticon](../blob/master/lib/tasks/panopticon.rake) you will need to change the status of the Smart Answer flow, or remove the Smart Answer flow entirely.
+
+As archiving tends not to be temporary, it's preferable to remove the flow entirely - and it can always be recovered from `git` history.
+
+If a flow needs to be removed urgently then it be can archived quickly by changing the flow's status to 'archived'. You should make sure work is scheduled to properly clean up the, now, unused flows.
+
+Once a flow has been removed the artefact will no longer register itself when the Smart Answers application is released. It will then to be archived in Panopitcon (as for other types of content), and a redirect added to `router-data` as a `prefix` route, so all parts of the Smart Answer URL are matched, not just the `slug` ([example](https://github.gds/gds/router-data/blob/master/data/slug-changes.csv#L623))
```

Comment:
Good spot, i've fixed the typo and rephrased it to be more explicit about requiring action, by YOU, the READER.

> You will then need to:
> 1. Archive the artefact in Panopticon (same process as other types of content)
> 2. Add a redirect `router-data` for the artefact slub. This should bea `prefix` route, so all parts of the Smart Answer URL are matched, not just the slug part of the ([This is a good example](https://github.gds/gds/router-data/blob/master/data/slug-changes.csv#L623))